### ISSUE TITLE
Show links with "bigbluebutton://" when the user is in a mobile device

### ISSUE THREAD

**Issue by daronco** [daronco/test-issue-migrate#41]
---

Author Name: **Leonardo Daronco** (@daronco)
Original Redmine Issue: 147, http://dev.mconf.org/redmine/issues/147

Original Assignee: Leonardo Daronco

---

@Google user: leonardo...@gmail.com@

All links to the BBB API use the HTTP protocol.
When a mobile client is accessing the website, the links should use "bigbluebutton://" instead of "http:// so that the BBB session will be opened in the native client and not in the browser.

Only links to API calls need to be changed. I believe that only bigbluebutton_rails will be affected, but maybe mconf-web will need some modifications as well.

**Comment by daronco** [comment]
---

Original Redmine Comment
Author Name: **Leonardo Daronco** (@daronco)
Original Date: 2011-05-16T03:00:00Z

---

@Google user: leonardo...@gmail.com@

**Comment by daronco** [comment]
---

Original Redmine Comment
Author Name: **Leonardo Daronco** (@daronco)
Original Date: 2011-05-20T03:00:00Z

---

@Google user: leonardo...@gmail.com@

We decided not to use "bigbluebutton://" as default when the client is on a mobile devise. We still use HTTP because the user might want to open the session in his browser (if he has a flash).

The "bigbluebutton://" links are now provided in a separate page (a dialog), that shows the link and also a QR code image that can be scanned by the mobile application to join the conference.

The feature was implemented in bigbluebutton_rails and still needs improvements (see "/p/mconf/issues/detail?id=95":Issue #95 ).
